pyo3-build-config/src/lib.rs

```diff
@@ -72,6 +71,8 @@ pub fn get() -> &'static InterpreterConfig {
             Ok(abi3_config())
         } else if impl_::cross_compile_env_vars().any() {
             InterpreterConfig::from_path(DEFAULT_CROSS_COMPILE_CONFIG_PATH)
+        } else if impl_::link_env_var_set() {
```

Comment:
To be honest I'd be tempted to try this first; it's exactly the configuration which was found by `pyo3-ffi` so it seems to make sense to use it if it exists.

Also, to avoid needing to read the environment twice, it would be cool to do something like

```rust
if let Some(parse_result) = InterpreterConfig::from_cargo_dep_env() {
    parse_result
} else if ...
```

Where `from_cargo_dep_env` returning `Option<Result<InterpreterConfig>>` should mean it doesn't need to panic.

Reply:
Thanks - this is smart!

I also appreciate the input on the ordering.